src/libraries/System.Xml.XmlSerializer/ref/System.Xml.XmlSerializer.cs

```diff
@@ -67,35 +67,39 @@ public partial class SoapAttributeAttribute : System.Attribute
+        [System.Diagnostics.CodeAnalysis.AllowNullAttribute]
         public string AttributeName { get { throw null; } set { } }
+        [System.Diagnostics.CodeAnalysis.AllowNullAttribute]
         public string DataType { get { throw null; } set { } }
-        public string Namespace { get { throw null; } set { } }
+        public string? Namespace { get { throw null; } set { } }
     }
     public partial class SoapAttributeOverrides
     {
         public SoapAttributeOverrides() { }
-        public System.Xml.Serialization.SoapAttributes this[System.Type type] { get { throw null; } }
-        public System.Xml.Serialization.SoapAttributes this[System.Type type, string member] { get { throw null; } }
-        public void Add(System.Type type, string member, System.Xml.Serialization.SoapAttributes attributes) { }
-        public void Add(System.Type type, System.Xml.Serialization.SoapAttributes attributes) { }
+        public System.Xml.Serialization.SoapAttributes? this[System.Type type] { get { throw null; } }
+        public System.Xml.Serialization.SoapAttributes? this[System.Type type, string member] { get { throw null; } }
+        public void Add(System.Type type, string member, System.Xml.Serialization.SoapAttributes? attributes) { }
+        public void Add(System.Type type, System.Xml.Serialization.SoapAttributes? attributes) { }
     }
     public partial class SoapAttributes
     {
         public SoapAttributes() { }
         public SoapAttributes(System.Reflection.ICustomAttributeProvider provider) { }
-        public System.Xml.Serialization.SoapAttributeAttribute SoapAttribute { get { throw null; } set { } }
-        public object SoapDefaultValue { get { throw null; } set { } }
-        public System.Xml.Serialization.SoapElementAttribute SoapElement { get { throw null; } set { } }
-        public System.Xml.Serialization.SoapEnumAttribute SoapEnum { get { throw null; } set { } }
+        public System.Xml.Serialization.SoapAttributeAttribute? SoapAttribute { get { throw null; } set { } }
+        public object? SoapDefaultValue { get { throw null; } set { } }
+        public System.Xml.Serialization.SoapElementAttribute? SoapElement { get { throw null; } set { } }
+        public System.Xml.Serialization.SoapEnumAttribute? SoapEnum { get { throw null; } set { } }
         public bool SoapIgnore { get { throw null; } set { } }
-        public System.Xml.Serialization.SoapTypeAttribute SoapType { get { throw null; } set { } }
+        public System.Xml.Serialization.SoapTypeAttribute? SoapType { get { throw null; } set { } }
     }
     [System.AttributeUsageAttribute(System.AttributeTargets.Field | System.AttributeTargets.Parameter | System.AttributeTargets.Property | System.AttributeTargets.ReturnValue)]
     public partial class SoapElementAttribute : System.Attribute
     {
         public SoapElementAttribute() { }
         public SoapElementAttribute(string elementName) { }
```

Comment:
wondering if this should be nullable arg considering property is allowing null

Reply:
Agree.